src/Parse/EnvFile.php

```diff
@@ -0,0 +1,238 @@
+<?php namespace  Winter\Storm\Parse;
+
+use Winter\Storm\Support\Str;
+use Winter\Storm\Parse\Contracts\DataFileInterface;
+
+/**
+ * Class EnvFile
+ */
+class EnvFile implements DataFileInterface
+{
+    /**
+     * @var array Lines of env data
+     */
+    protected $env = [];
+
+    /**
+     * @var array Map of variable names to line indexes
+     */
+    protected $map = [];
+
+    /**
+     * @var string|null Filepath currently being worked on
+     */
+    protected $filePath = null;
+
+    /**
+     * EnvFile constructor
+     */
+    public function __construct(string $filePath)
+    {
+        $this->filePath = $filePath;
+
+        list($this->env, $this->map) = $this->parse($filePath);
```

Comment:
```suggestion
        [$this->env, $this->map] = $this->parse($filePath);
```

Reply:
I personally prefer using list() rather than just square brackets, easier to google what the heck's happening for new devs.

Reply:
Fair point.